### ISSUE TITLE
Why you did not use SensorEvent.timestamp?

### ISSUE THREAD

**Issue by ftabashir** [maddevsio/mad-location-manager#77]
In `KalmanLocationService.onSensorChanged` elapsed nanos was retrieved like this:
```
long now = android.os.SystemClock.elapsedRealtimeNanos();
```
While we have it already in `SensorEvent.timestamp`.

Is there any issue in using `SensorEvent.timestamp` which you avoid to use?

**Comment by Lezh1k** [comment]
Yes. I'm not sure why, but seems, like those events use different clock sources. 
I wanted to be sure, that we use same clock source for each event type.
